roaringbitmap/src/main/java/org/roaringbitmap/longlong/Roaring64NavigableMap.java

```diff
@@ -1602,6 +1908,28 @@ public boolean equals(Object obj) {
     return Objects.equals(highToBitmap, other.highToBitmap);
   }
 
+  /**
+   * clone current Roaring64NavigableMap instance, the new cloned instance will have
+   * the same serialization mode with this one
+   */
+  @Override
```

Comment:
I note this .clone will handle only 64-b bitmaps smaller than 2G.

Reply:
yes, it uses `ByteArrayOutputStream` and `ByteArrayInputStream` as a transit layer.
option 1: document this limit <2GB, and throw exception, which looks like `Roaring64Bitmap.clone`
option 2: impl a new outputsteam/inputstream that supports >=2GB. (maybe use `DirectByteBuffer` as a backend)

i prefer option 1, which one do you prefer? or do you have other suggestions? thanks @blacelle